app/styles/ui/dialogs/_release-notes.scss

```diff
@@ -57,7 +57,7 @@
     flex-direction: row;
     padding-left: var(--spacing-triple);
     padding-right: var(--spacing-triple);
-    overflow: auto;
+    overflow: scroll hidden;
```

Comment:
can we make this one `auto hidden`? that means it will only show scroll bars on the release notes when they are needed. thanks @cwongmath!